docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -211,23 +226,14 @@ Example
 2. Model fitting
 ================
 
-The reduction of a block :math:`x_0` from :math:`X_0` to the corresponding block :math:`x_1` from :math:`X_1` is accomplished by the matrix multiplication :math:`x_0 B = x_1`, where :math:`B` is a coefficient matrix of size :math:`m \times K`, where :math:`m` is the number of columns in block :math:`x_0` and :math:`K` is the number of alpha values used in the reduction. As an added wrinkle, if the ridge reduction is being performed against multiple phenotypes at once, each phenotype will have its own :math:`B`, and for convenience we panel these next to each other in the output into a single matrix, so :math:`B` in that case has dimensions :math:`m \times (K \times P)` where :math:`P` is the number of phenotypes. Each matrix :math:`B` is specific to a particular block in :math:`X_0`, so the Spark DataFrame produced by the ``RidgeReducer`` can be thought of matrices :math:`B` from all the blocks, one stacked atop another.
-
-Parameters
-----------
-
-- ``block_df``: Spark DataFrame representing the beginning block matrix
-- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models
-- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs
-- ``covariate_df``: Pandas DataFrame containing covariates to be included in every model in the stacking
-  ensemble (optional)
+Reduction model fitting is performed calling the ``RidgeReduction.fit()`` function (no parameters needed). The reduction of a block :math:`x_0` from :math:`X_0` to the corresponding block :math:`x_1` from :math:`X_1` is accomplished by the matrix multiplication :math:`x_0 B = x_1`, where :math:`B` is an estimated coefficient matrix of size :math:`m \times K`, where :math:`m` is the number of columns in block :math:`x_0` and :math:`K` is the number of alpha values used in the reduction. As an added wrinkle, if the ridge reduction is being performed against multiple phenotypes at once, each phenotype will have its own :math:`B`, and for convenience we panel these next to each other in the output into a single matrix, so :math:`B` in that case has dimensions :math:`m \times (K \times P)` where :math:`P` is the number of phenotypes. Each matrix :math:`B` is specific to a particular block in :math:`X_0`, so the Spark DataFrame produced by the ``RidgeReduction.fit()`` can be thought of matrices :math:`B` from all the blocks, one stacked atop another.
```

Comment:
Wording nit: `Reduction model fitting is performed calling` -> `Perform reduction model fitting by calling`